02/app.js

```diff
@@ -1,28 +1,34 @@
-document.addEventListener('DOMContentLoaded', init);
+document.addEventListener("DOMContentLoaded", init);
 
 function init() {
-    const clickEl = document.querySelector('.error--click');
-    const enterEl = document.querySelector('.error--enter');
+  const clickEl = document.querySelector(".error--click");
+  const enterEl = document.querySelector(".error--enter");
 
-    setRandomPosition(clickEl);
-    setRandomPosition(enterEl);
-
-    initEventWithError(clickEl, 'click', new RangeError('Błąd zakresu!'));
-    initEventWithError(enterEl, 'mouseenter', new TypeError('Błąd typu!'));
+  setRandomPosition(clickEl);
+  setRandomPosition(enterEl);
 
+  initEventWithError(clickEl, "click", new RangeError("Błąd zakresu!"));
+  initEventWithError(enterEl, "mouseenter", new TypeError("Błąd typu!"));
 }
 
 function setRandomPosition(element, error = null) {
-    element.style.top = Math.random() * 600 + 'px';
-    element.style.left = Math.random() * 800 + 'px';
+  try {
```

Comment:
Przechwytywanie błędu powinno być gdzieś "niżej", bo w inny przypadku można by było napisać prosty `if`.
W tym przypadku chodzi o to, aby `try` obejmowało `seRandomPosition`